tests/fullstack/fullstack.go

```diff
@@ -0,0 +1,207 @@
+package fullstack
```

Comment:
Inspired by the original PR, but I had to fix a bunch of things reg things that happened since then; but the essence is the same.